doc/book/connectors/__python.rst

```diff
@@ -39,8 +39,221 @@ For an example of using Python API with
+        -   Github Actions
+        -   No (tests exists)
+        -   No (tests exists)
+        -   Travis CI and Appveyor
+
+    *   -   Github Stars
+        -   51
+        -   38
+        -   17
+        -   78
+
+    *   -   Static Analysis
+        -   Yes (Flake8)
+        -   No
+        -   No
+        -   No
+
+    *   -   Packaging
+        -   `Pip <https://pypi.org/project/asynctnt/>`__
+        -   `Pip <https://pypi.org/project/aiotarantool/>`__
+        -   `Pip <https://pypi.org/project/gtarantool/>`__
+        -   `Deb, RPM, Pip <https://github.com/tarantool/tarantool-python#download-and-install>`__
+
+    *   -   Code coverage
+        -   Yes
+        -   No
+        -   No
+        -   Yes
+
+    *   -   Support asynchronous mode
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes (`gevent
+            <https://www.gevent.org/api/gevent.event.html#gevent.event.AsyncResult>`__,
+            example: `test_gevent.py
+            <https://github.com/shveenkov/gtarantool/blob/master/tests/test_gevent.py>`__)
+        -   No
+
+    *   -   Batching support
+        -   No
+        -   No
+        -   No
+        -   No (`issue #55 <https://github.com/tarantool/tarantool-python/issues/55>`__)
+
+    *   -   Schema reload, space and index names
+        -   Yes (*auto_refetch_schema*)
+        -   Yes (handle *SchemaReloadException*)
+        -   No
+        -   Yes (handle *SchemaReloadException*)
+
+    *   -   Schema reload, Access tuple fields by names
+        -   Yes
+        -   FIXME
+        -   FIXME
+        -   FIXME
+
+    *   -   `SQL support <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`__
+        -   Yes (tests/test_op_sql.py)
+        -   No (using call server-side functions)
+        -   No
+        -   Yes (tarantool/connection.py)
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`__
+        -   No
+        -   No
+        -   No
+        -   No (`issue #163 <https://github.com/tarantool/tarantool-python/issues/163>`__)
+
+    *   -   `Varbinary support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `UUID support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `Decimal support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   Almost (v2 branch)
+        -   No
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR support <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `[Soon] <https://github.com/tarantool/tarantool/discussions/6244>`__ Datetime support
```

Comment:
I would get rid of 'Soon' marks on this page.

Reply:
Without mark, it looks as existed Tarantool's feature. Okey, removed "Soon" marks.